webapp/views/wpt-results.js

```diff
@@ -861,6 +879,41 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     this.path = this.searchResults[(n + next) % n].test;
   }
 
+  sortTestname() {
+    if (!this.displayedNodes) {
+      return;
+    }
+    const sortedNodes = this.displayedNodes.slice();
+    sortedNodes.sort(function (a, b) {
+      if (a.path < b.path) {
+        return -1;
+      }
+
+      if (a.path > b.path) {
+        return 1;
+      }
+      return 0;
+    })
+
+    this.displayedNodes = sortedNodes;
+  }
+
+  sortTestResults() {
+    const [index] = arguments;
```

Comment:
I thought of some possible changes/suggestions after testing the UI:

- Displaying which row is currently sorted by visually changing the sort arrow (slight color change?)
- (Reverse sorting) Keeping track of what row the data is sorted by and reversing that order if the user selects the row again would feel intuitive and could have additional use. Also changing the arrow to point upward in the case of reverse sorting as an obvious visual cue of what is happening.
- The sort row feels like it should have column labels of some sort, but those column names are essentially in the row above them. I'm not sure the best approach here, but it's something that felt odd when I first looked at it.

Nice work @KyleJu!